src/main/java/net/dv8tion/jda/internal/utils/InvalidTokenException.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2015 Austin Keener, Michael Ritter, Florian Spieß, and the JDA contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.dv8tion.jda.internal.utils;
+
+/**
+ * Indicates that an invalid token was given when trying to login the Discord API
+ *
+ * @author java-coding-prodigy
```

Comment:
Remove the `@author`, see the `Usage of author annotation documentation` thread

https://discord.com/channels/125227483518861312/941340956546977802/941383460344455248